Tests/SentryTests/Integrations/ANR/SentryANRTrackingIntegrationTests.swift

```diff
@@ -68,37 +53,29 @@ class SentryANRTrackingIntegrationTests: XCTestCase {
         assertArrayEquals(expected: expexted, actual: Array(options.enabledIntegrations))
     }
     
-    func testANRDetected_UpdatesAppStateToTrue() {
+    func testANRDetected_EventCaptured() {
         givenInitializedTracker()
         
         Dynamic(sut).anrDetected()
         
-        guard let appState = fixture.fileManager.readAppState() else {
-            XCTFail("appState must not be nil")
-            return
-        }
-
-        XCTAssertTrue(appState.isANROngoing)
-    }
-    
-    func testANRStopped_UpdatesAppStateToFalse() {
-        givenInitializedTracker()
-        
-        Dynamic(sut).anrStopped()
-        
-        guard let appState = fixture.fileManager.readAppState() else {
-            XCTFail("appState must not be nil")
-            return
+        assertEventWithScopeCaptured { event, _, _ in
+            XCTAssertNotNil(event)
+            guard let ex = event?.exceptions?.first else {
+                XCTFail("ANR Exception not found")
+                return
+            }
+            
+            XCTAssertEqual(ex.mechanism?.type, "anr")
+            XCTAssertEqual(ex.type, "App Hanging")
+            XCTAssertEqual(ex.value, "Application Not Responding for at least 4500 ms.")
```

Comment:
`m`: Can we please assert that all threads with stacktraces are attached. We don't need to validate these, but just to make sure they are there.